x-pack/plugins/observability_solution/logs_data_access/server/services/get_logs_rate_timeseries/get_logs_rate_timeseries.ts

```diff
@@ -104,12 +104,10 @@ export function createGetLogsRateTimeseries() {
 
     return buckets
       ? buckets.reduce<LogsRateTimeseriesReturnType>((acc, bucket) => {
-          const totalCount = bucket.doc_count;
-
           const timeseries = bucket.timeseries.buckets.map((timeseriesBucket) => {
             return {
               x: timeseriesBucket.key,
-              y: timeseriesBucket.doc_count / totalCount,
```

Comment:
We were dividing by the total because log rate should be a % value.

I think this is right, what we need to fix instead is the client side

Reply:
But we do have this -> https://github.com/elastic/kibana/issues/188247, why do you say it should be a %?

Reply:
Hi @roshan-elastic ,  

It seems we’re unclear on the intention behind removing the % in https://github.com/elastic/kibana/issues/188247. 
The % was removed from the numeric format but kept in the calculations, so the chart still shows percentage data but without being multiplied by 100 or displaying % as the unit.  

As a result, the Y-axis values and tooltips aren’t displaying correctly. From my understanding, the fix would be to remove the percentage from the calculations, aligning it with the formula we show users.  

Could you confirm if the solution in this PR is correct and that we want the chart to show only the number of logs corresponding to the X-axis values, not percentages? If that’s not the case, could you please clarify what we actually want to display in this chart?  

Thanks!

Reply:
Hey @iblancof,

Assuming that the 'Log Error Rate' is also a 'rate' (as you've shown for the screenshot for the 'Log Rate' - then LGTM!

(i.e. there will be no %s anymore)